code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1028,6 +1028,17 @@
 	log_combat(user, L, "took out a blood contract on", src)
 	qdel(src)
 
+/obj/item/gun/ballistic/revolver/doublebarrel/super
```

Comment:
can you make this fit onto the H.E.C.K suit?

Reply:
none of the good lavaland armor holds all gun subtypes so if they want to not be holding it at all times they have to trade for weaker sec armor is the idea